framework/tests/value/MinLenLUB.java

```diff
@@ -23,10 +22,8 @@ public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
         } else {
             arr = arg2;
         }
-        // :: error: (assignment.type.incompatible)
```

Comment:
Same issue here: I think we need to test LUIB of `@BottomVal` and `@MinLen(4)`, and LUB of `@MinLen(10)` and `@MinLen(4)`.